README.md

```diff
@@ -5,7 +5,7 @@ Download [Firefox iOS](https://apps.apple.com/app/firefox-web-browser/id98980492
 <table>
   <tr>
     <th style="border: none;"><strong>Firefox iOS</strong></th>
-    <td style="border: none;"><img src="https://img.shields.io/badge/Xcode-15.4-blue?logo=Xcode&logoColor=white" alt="Firefox-iOS"></td>
+    <td style="border: none;"><img src="https://img.shields.io/badge/Xcode-16.0-blue?logo=Xcode&logoColor=white" alt="Firefox-iOS"></td>
```

Comment:
😭 Sorry, Firefox iOS currently does not support 16.0.

Reply:
Ah ok thanks @clarmso, i was updating that with the pipeline, i found i was grepping the wrong stack in the Bitrise file. I was taking the NewXcodeVersion pipeline stack. Now it should be good i've updated the readme and the pipeline